### PR TITLE
Fix for fancy checkbox in firefox + improving main content css + add page templates for docs site

### PR DESCRIPTION
**Pull request by bitterdev**
While cross browser testing the theme i have seen that the checkboxes are not rendered. This pr is fixing that issue.

### CODE REVIEW

**Comment by aembler** [comment]
Can you pull down a copy of master and redo this pull request? It has merge conflicts and I assume it's because you need to do a git pull or rebase first

**Comment by bitterdev** [comment]
Not this should work fine.

File: data.xml [hunk 1]
```diff
@@ -13,7 +13,129 @@
                       category="user">
             <type placeholder=""/>
         </attributekey>
-
+        <attributekey handle="profile_private_messages_enabled" name="Private Messages" package="concrete_cms_theme"
```
Comment (aembler) [inline data.xml:16]: I don't think these attributekeys should be here. Shouldn't this have been moved into the community package?

**Review by aembler — CHANGES_REQUESTED** [review]

File: data.xml (hunk 1 above)
Comment (bitterdev) [reply data.xml:16]: Okay i will move this to the community package.

**Comment by bitterdev** [comment]
Removed this section. No need to copy this to the community repo because it's already there (see data.xml)
I have also added some css rules that are making the main content looking nicer (useful for the docs site)

**Comment by bitterdev** [comment]
Added page templates for documentation site

**Pull request merged**